### DISCUSSION TITLE
Coastal flood warning

### DISCUSSION THREAD

**Discussion by mmickelson-verisk** [weather-gov/api#456] [Q&A] [answered]
Hi there,

First off if this is not the correct place for this question I apologize, please let me know.

I have been using the alert api successfully for a few event types (flood mainly). 

I haven't seen any data come back from either of these two types (Coastal Flood Warning, Hurricane Warning) and was hoping to get a verification that this is expected.

https://api.weather.gov/alerts/active?status=actual&event=Coastal%20Flood%20Warning,Hurricane%20warning

When I didn't get information for an event in the news (Block Island off the coast of Long Island flooded Aug 23). I wondered if something were wrong.

Thanks

**Comment by jah1007** [comment, marked as answer]
Based on what I am seeing there was a Hurricane Warning that should have showed up in the feed up until the evening of August 22nd. It was cancelled at that time and replaced by the Weather Forecast Office Boston/Norton with a Tropical Storm Warning. They never issued a Coastal Flood Warning for the event. Only Coastal Flood Advisories were issued.

**Reply by mmickelson-verisk** [reply]
This is good to know. Thank you. Is there a way for me to get that information through the api?

**Reply by scadergit** [reply]
You can review the history of all alerts, not just active, to review the event chain. You have to know how chaining of alerts works, which isn't always straight forward, but all events of a chain will have references to what it replaced or what it was replaced by.

**Reply by mmickelson-verisk** [reply]
Ok I can look into that. Thanks for the help.
